.github/workflows/sync-documentation.yml

```diff
@@ -0,0 +1,30 @@
+name: Sync-Documentation
+
+on:
+  schedule:
+    - cron:  '0 3 * * *'
+  pull_request:
+    branches: [ main ] 
+
+jobs:          
+  update-documentationmd:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Update documentation
+        id: vars
+        run: make refresh-docs
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: |
+            sync documentation with containerd/containerd
+
+            Signed-off-by: Scott Haddlesey <haddscot@amazon.com>
```

Comment:
Bot signatures should be distinguishable from real signatures

Reply:
I based this on sync-releases: https://github.com/containerd/containerd.io/blob/main/.github/workflows/sync-releases.yml#L28

what signature should I use for bot commits?